
Generate adjacency matrix:    : 1367it [00:01, 1300.66it/s]
Training:    :   0%|                                                                                                                                                                                                | 0/125 [00:00<?, ?it/s]
Training:    :   0%|                                                                                                                                                                                                | 0/125 [00:01<?, ?it/s]
Traceback (most recent call last):
  File "run.py", line 122, in <module>
    KTtrain()
  File "run.py", line 94, in KTtrain
    model, optimizer = eval.train_epoch(model, trainLoaders[0], optimizer, loss_func)
  File "C:\Users\15737\OneDrive\桌面\DGEKT\KnowledgeTracing\evaluation\../..\KnowledgeTracing\evaluation\eval.py", line 82, in train_epoch
    logit_c, logit_t, logit_ensemble = model(batch)
  File "C:\Users\15737\AppData\Roaming\Python\Python38\site-packages\torch\nn\modules\module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "C:\Users\15737\OneDrive\桌面\DGEKT\KnowledgeTracing\evaluation\../..\KnowledgeTracing\model\Model.py", line 133, in forward
    emseble_logit = self.fc_ensemble((1/3)*torch.cat([out_d, out_h, out_light], -1))
  File "C:\Users\15737\AppData\Roaming\Python\Python38\site-packages\torch\nn\modules\module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "C:\Users\15737\AppData\Roaming\Python\Python38\site-packages\torch\nn\modules\linear.py", line 114, in forward
    return F.linear(input, self.weight, self.bias)
RuntimeError: mat1 and mat2 shapes cannot be multiplied (6400x384 and 256x3162)